[2025-05-28T02:33:46.100+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-28T02:33:46.151+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bash_dag.print_date scheduled__2025-05-20T00:00:00+00:00 [queued]>
[2025-05-28T02:33:46.169+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bash_dag.print_date scheduled__2025-05-20T00:00:00+00:00 [queued]>
[2025-05-28T02:33:46.170+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-05-28T02:33:46.227+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): print_date> on 2025-05-20 00:00:00+00:00
[2025-05-28T02:33:46.253+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1503) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-28T02:33:46.260+0000] {standard_task_runner.py:72} INFO - Started process 1507 to run task
[2025-05-28T02:33:46.254+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'bash_dag', 'print_date', 'scheduled__2025-05-20T00:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/Sesion2/Ejemplo3.py', '--cfg-path', '/tmp/tmpxzcs6qko']
[2025-05-28T02:33:46.265+0000] {standard_task_runner.py:105} INFO - Job 8: Subtask print_date
[2025-05-28T02:33:46.313+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-28T02:33:46.421+0000] {task_command.py:467} INFO - Running <TaskInstance: bash_dag.print_date scheduled__2025-05-20T00:00:00+00:00 [running]> on host 31d54648d280
[2025-05-28T02:33:46.424+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-28T02:33:47.079+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='bash_dag' AIRFLOW_CTX_TASK_ID='print_date' AIRFLOW_CTX_EXECUTION_DATE='2025-05-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-20T00:00:00+00:00'
[2025-05-28T02:33:47.080+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-28T02:33:47.108+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-05-28T02:33:47.109+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'date']
[2025-05-28T02:33:47.185+0000] {subprocess.py:86} INFO - Output:
[2025-05-28T02:33:47.392+0000] {subprocess.py:93} INFO - Wed May 28 02:33:47 UTC 2025
[2025-05-28T02:33:47.394+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-05-28T02:33:47.519+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-28T02:33:47.519+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=bash_dag, task_id=print_date, run_id=scheduled__2025-05-20T00:00:00+00:00, execution_date=20250520T000000, start_date=20250528T023346, end_date=20250528T023347
[2025-05-28T02:33:47.891+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-28T02:33:48.080+0000] {taskinstance.py:3900} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2025-05-28T02:33:48.085+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
